packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -47,6 +51,17 @@ const TypographyMaxWidth = styled<TypographyComponent>(Typography)`
   max-width: 300px;
 `;
 
+const TableComponent = styled(RawTable)`
+  width: 100%;
+  table-layout: fixed;
+  td:first-child {
+    border-right: 1px solid ${({ theme }) => theme.colors.neutral150};
+  }
+`;
+const TableCell = styled(Td)`
+  padding: 16px !important;
+`;
```

Comment:
I prefer we avoid the important by overriding the `td:first-of-type` rule that is set on each `tr`. I believe that should give the same result. You won't need the `TableCell` styled component anymore and you can use `Td` directly

```suggestion
  td:first-child {
    border-right: 1px solid ${({ theme }) => theme.colors.neutral150};
  }
  td:first-of-type {
    padding: ${({ theme }) => theme.spaces[4]};
  }
`;
```

Reply:
Ohh right, I see what went wrong earlier! I had mistakenly applied padding to `first-child` instead of `first-of-type`, which is why it didn’t work as expected so I used `!important` for padding. Thanks for the suggestion!